### PR TITLE
Feature/match details layout

### PR DESCRIPTION
**Pull request by jorgearguellles**
I work in the layout but is not ended ...

@dasafodev @pabloverduzco @MaoRguez 
And ... I don't know whay background-color: var(--linear); is't worked!

### CODE REVIEW

**Comment by ghost** [comment]
Web capture_11-3-2021_201733_luxxon-git-feature-matchdetailslayout-luxxon vercel app

Something got broken in the authentication page, what do you think happened?

File: app/components/login_layout/login_layout.module.css [hunk 1]
```diff
@@ -1,9 +1,8 @@
 .container {
   display: grid;
   grid-template-columns: 1fr 1fr;
-  height: calc(100vh - 70px);
   width: 100vw;
-  background-image: var(--gradient-image), url('/images/background.webp');
```
Comment (ghost) [inline app/components/login_layout/login_layout.module.css:6]: I think this is why at the authentication page the background is white.

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/components/login_layout/login_layout.module.css (hunk 1 above)
Comment (jorgearguellles) [reply app/components/login_layout/login_layout.module.css:6]: I found it! and fix it in local.
@pabloverduzco  Do you have been fixed? or Can I push...?

Comment (ghost) [reply app/components/login_layout/login_layout.module.css:6]: Go ahead, send the fix! 😄

**Review by dasafodev — APPROVED** [review]

**Pull request merged**